xbmc/pvr/PVRGUIActions.h

```diff
@@ -366,6 +366,15 @@ namespace PVR
     */
     bool DeleteTimer(const CFileItemPtr &item, bool bIsRecording, bool bDeleteRule) const;
 
+    /*!
+     * @brief Delete a timer or timer rule, never showing a confirmation dialog.
```

Comment:
This doesn't seem right since there's a call to `HELPERS::ShowYesNoDialogText()` in the method

Reply:
Yeah, good catch. Needs update.